### PR TITLE
[4] show_total_votes in vote plugin

### PR DESCRIPTION
**Pull request by PhilETaylor**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/33137

### Summary of Changes

Implement total number of votes display

### Testing Instructions

Enable "Content - Vote" plugin and configure the new option to SHOW the "Show total number of votes"
Content -> options -> enable voting
View article on the frontend

### Actual result BEFORE applying this Pull Request

No visible indication of how many votes cast

### Expected result AFTER applying this Pull Request

Screenshot 2021-04-15 at 19 56 16
Screenshot 2021-04-15 at 19 56 11

### Documentation Changes Required

none - screenshots?

### CODE REVIEW

File: administrator/language/en-GB/plg_content_vote.ini [hunk 1]
```diff
@@ -10,6 +10,8 @@ PLG_VOTE_POSITION_LABEL="Position"
 PLG_VOTE_RATE="Rate"
 PLG_VOTE_STAR_RATING="Rating: %s out of 5"
 PLG_VOTE_TOP="Top"
+PLG_VOTE_TOTAL_VOTES="Total Votes: %s"
+PLG_VOTE_TOTAL_VOTES_LABEL="Show total number of votes"
```
Comment (brianteeman) [inline administrator/language/en-GB/plg_content_vote.ini]:
```suggestion
PLG_VOTE_TOTAL_VOTES_LABEL="Number of Votes"
```

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on f3353dc7be0786e5d4840de0dc4e58eaf2fbae3c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33143">issues.joomla.org/tracker/joomla-cms/33143</a>.</sub>

**Comment by sandewt** [comment]
I have tested this item :white_check_mark: successfully on f3353dc7be0786e5d4840de0dc4e58eaf2fbae3c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33143">issues.joomla.org/tracker/joomla-cms/33143</a>.</sub>

**Comment by chmst** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33143">issues.joomla.org/tracker/joomla-cms/33143</a>.</sub>

**Pull request merged**